fmn/consumer/backends/sse.py

```diff
@@ -130,7 +131,7 @@ def __init__(self, host='localhost',
                  queue_name='skrzepto.id.fedoraproject.org',
                  expire_ms=1*60*60*1000,
                  port=5672,
-                 log=None):
+                 log=logging.getLogger("fmn")):
```

Comment:
It's likely better to keep None in the function declaration and check lower down ``if log is None``...